client/src/django.ts

```diff
@@ -77,11 +88,15 @@ const djangoClient = {
     const { data } = await apiClient.get(`/projects/${projectId}/task_overview`);
     return data;
   },
-  async settings(projectId: string): Promise<Settings> {
+  async settings(projectId: string): Promise<ProjectSettings> {
     const { data } = await apiClient.get(`/projects/${projectId}/settings`);
     return data;
   },
-  async setSettings(projectId: string, settings: Settings) {
+  async setSettings(projectId: string, settings: ProjectSettings) {
+    if (projectId === 'global') {
+      const resp = await apiClient.put('/global/settings', settings);
+      return resp.status === 200 ? resp.data : null;
```

Comment:
Seems like we want an `else` after this, no? Or better yet, break it up into `setGlobalSettings` and `setProjectSettings`

Reply:
changed in f932555